rtgui/controllines.cc

```diff
@@ -260,7 +267,24 @@ bool ControlLineManager::drag1(int modifierKey)
         autoSetLineType(selected_object);
     }
 
-    return false;
+    return true;
+}
+
+void ControlLineManager::releaseEdit(void)
```

Comment:
@Lawrence37 If you're touching code, you're free to remove artifacts like this prehistoric `void` parameter.